src/main/java/milk/manager/iot/Writer.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * @author @liosyk
+ * Project name: JavaLab234
+ * Package name: milk.manager.iot
+ * Class: Writer
+ **/
+
+package milk.manager.iot;
+
+import milk.products.iot.MilkProduct;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+
+public final class Writer {
+    //1000-7
+    private Writer() {
+
+    }
+    public static void writeToCSV(final List<MilkProduct> milkbox) throws IOException {
+        try (FileWriter writer = new FileWriter("result.csv", StandardCharsets.UTF_8)) {
+            String previousClassName = "";
+
+            for (MilkProduct box: milkbox) {
+                if (!box.getClass().getSimpleName().equals(previousClassName)) {
+                    writer.write("\r\n");
+                    writer.write(box.getHeaders());
+                    writer.write("\r\n");
+                    previousClassName = box.getClass().getSimpleName();
+                }
+                writer.write(box.toCSV());
+                writer.write("\r\n");
+            }
+        }
+    }
+
```

Comment:
Pay attention to code style conventions.
Consider **removing** redundant blank lines.